src/games/strategy/triplea/ai/proAI/ProNonCombatMoveAI.java

```diff
@@ -249,7 +249,7 @@ private void findUnitsThatCantMove(final Map<Territory, ProPurchaseTerritory> pu
 
       // Add max defenders that can be purchased to each territory
       for (final Territory t : moveMap.keySet()) {
-        if (ProMatches.territoryHasInfraFactoryAndIsNotConqueredOwnedLand(player, data).match(t)) {
```

Comment:
It actually is used in a bunch of places still otherwise I probably would have just updated it to the new one.